src/main/kotlin/dev/tmsoft/lib/params/sorting/Sorting.kt

```diff
@@ -0,0 +1,63 @@
+package dev.tmsoft.lib.params.sorting
+
+import dev.tmsoft.lib.params.Field
+import dev.tmsoft.lib.params.PathValues
+import dev.tmsoft.lib.params.SingleValue
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.EnumerationNameColumnType
+import org.jetbrains.exposed.sql.Query
+import org.jetbrains.exposed.sql.SortOrder
+import org.jetbrains.exposed.sql.Table
+
+abstract class Sorting(val table: Table) {
+    private val fields: MutableList<Field> = mutableListOf()
+
+    fun fields(): List<Field> = fields
+
+    fun add(
+        name: String,
+        column: Column<*>? = null,
+        possibleValues: List<String>? = null
+    ): Field {
+        val tableColumn = column ?: table.columns.first { it.name == name }
+
+        val field = Field(
+            name,
+            { values ->
+                val sortOrder = (values.single() as SingleValue).value
+                    .replace(" ","")
```

Comment:
Maybe this `replace` is not needed?

Reply:
I think it give more path options. Because SortOrder.valuesOf("desc ") with whitespace doesn't work

Reply:
I think we should have method trim()